src/coreComponents/finiteElement/elementFormulations/H1_Hexahedron_Lagrange1_GaussLegendre2.hpp

```diff
@@ -80,11 +80,8 @@ class H1_Hexahedron_Lagrange1_GaussLegendre2 final : public FiniteElementBase
 
   /// The number of quadrature points per element.
   constexpr static localIndex numQuadraturePoints = 8;
-
   ///
-  constexpr static int numSamplingPoints = 1000;  // 10 in each direction.
-
-  constexpr static int numSamplingPointsPerDirection = 10;
+  constexpr static int numSamplingPoints = numSamplingPointsPerDirection * numSamplingPointsPerDirection * numSamplingPointsPerDirection;
```

Comment:
You may want to define a function like:
```c++
template <typename T>
constexpr T pow(T x, unsigned int n)
{
   return n == 0 ? 1 : x * pow(x, n-1);
}
```
(You'll need to find a different name to not conflict.)